QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,179 @@
+
+# `libyaml_vendor` Quality Declaration
+
+The package `libyaml_vendor` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`libyaml_vendor` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`libyaml_vendor` is at or above a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+This is a vendor package for `libyaml` and as such does not declare its own API.
+
+### API Stability Policy [1.iv]
+
+`libyaml_vendor` does not have an API of its own, and will limit its dependency on `libyaml` to API stable releases within a released ROS distribution.
+
+### ABI Stability Policy [1.v]
+
+`libyaml_vendor` does not have an ABI of its own, and will limit its dependency on `libyaml` to ABI stable releases within a released ROS distribution.
+
+### ABI and ABI Stability Within a Released ROS Distribution [1.vi]
+
+As `libyaml_vendor` does not come with its own API/ABI, changes will not affect API/ABI stability.
+
+## Change Control Process [2]
+
+`libyaml_vendor` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process).
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](./CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
+
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/libyaml_vendor/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/libyaml_vendor/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/libyaml_vendor/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/libyaml_vendor/)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`libyaml_vendor` does not currently document its features.
```

Comment:
add here some details why this package doesn't have documentation. Does it have any feature? It's just compiling and installing  an external library.

Reply:
Mmm I think the vendor packages will need features documentation, at least a brief summary. @brawner mind giving your thoughts on this?

Reply:
Yah, it's unclear what features it would document. Maybe it's enough if it just points to libyaml's features in the README?

Reply:
I'll ask to the team, I think so far we have stated for vendored packages that there is no features documentation available. So this shouldn't stop merging this as it states the current state.

Reply:
I added an extra explanation as to why this vendor package does not require features documentation.